inginious/common/base.py

```diff
@@ -55,6 +55,14 @@ def get_json_or_yaml(file_path, content):
         return inginious.common.custom_yaml.dump(content)
 
 
+def hash_content(content):
```

Comment:
This should have a more explicit name (``hash_password`` for instance), and be located in the frontend files. This whole file probably shouldn't be modified.